packages/service/src/service.ts

```diff
@@ -126,7 +126,7 @@ export class ThreeIDService  {
     try {
       // Skip migration if muport or known failure
        // @ts-ignore
-      did = await manage.createAccount({ legacyDid, skipMigration: Boolean(muportDid || willFail) })
+      did = await manage.createAccount({ legacyDid, skipMigration: Boolean(muportDid || willFail), legacyDid })
```

Comment:
Why is legacyDid added to this object twice?
```suggestion
      did = await manage.createAccount({ legacyDid, skipMigration: Boolean(muportDid || willFail) })
```